src/main/java/nextstep/subway/domain/vo/Sections.java

```diff
@@ -23,6 +23,10 @@ public class Sections {
     public Sections() {
     }
 
+    public List<Section> getValue() {
+        return sections;
+    }
```

Comment:
방어적 복사를 위해 불변 컬렉션으로 반환해도 좋을 것 같아요 😃 
```suggestion
    public List<Section> getValue() {
        return Collections.unmodifiableList(sections);
    }
```